webapp/blueprint.py

```diff
@@ -0,0 +1,95 @@
+import flask
+
+
+def jp_website_blueprint():
+    """
+    Return a blueprint for all routes of the main jp.ubuntu.com website
+    """
+    jp_website = flask.Blueprint(
+        "jp_website",
+        __name__,
+        template_folder="/templates",
+        static_folder="/static",
+    )
+
+    @jp_website.route("/")
+    def homepage():
+        status_code = 200
```

Comment:
no need to add var, by default `flask.render_template("")` returns a status code 200

It's usefule to have a var like this if you are doing API calls that could change the state of the page.

Reply:
(it can be removed on every function)